chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -141,9 +140,29 @@ object CBPurchase {
         sharedInstance(context).restorePurchases(completionCallback)
     }
 
-    /* Chargebee Method - used to validate the receipt of purchase  */
+    /**
+     * This method will be used to validate the receipt with Chargebee.
+     * In case of failed syncing with Chargebee during the purchase flow
```

Comment:
`This method can be used to validate the receipt with Chargebee, when syncing with Chargebee fails after the successful purchase`